docs/azure/remote-debugging.md

```diff
@@ -8,3 +8,39 @@ MetaDescription: Azure Remote Debugging with Visual Studio Code
 DateApproved:
 ---
 # Azure Remote Debugging
+
+Connect the Visual Studio Code debugger to your apps running on **Azure App Service** on **Linux**. The debugger works the same as when it's connected to a local Node.js process - including the use of Breakpoints and Logpoints.
```

Comment:
Do we want to make this specific to Node.js for now? "Azure Remote Debugging" is quite generic.

Reply:
Changed it to "Azure Remote Debugging for Node.js Apps"